source/_docs/modules-plugins-known-issues.md

```diff
@@ -69,6 +69,27 @@ If you have already enabled the Apache Solr Multilingual module and found that y
 $baseUrl = '/ckfinder/userfiles/';
 ```
 
+**Issue (CKFinder 3)**: CKFinder's working folder cache is located inside the module's folder hitting this error `Fatal error: Uncaught CKSource\CKFinder\Exception\AccessDeniedException: Couldn't create resource type directory. Please check permissions.`
+
+**Solution (CKFinder 3)**: Manually edit the `ckfinder/config.php` file and edit the following lines to point to the standard writeable path. Please note that you will need to create the folder `ckfinder-cache` under `sites/default/files` for this to work:
```

Comment:
Would a change like this be overwritten when the plugin is updated? We should try to avoid workarounds that involve hacking the module itself.